include/common/qrack_types.hpp

```diff
@@ -13,19 +13,22 @@
 #pragma once
 
 #include <memory>
+#include <random>
 
 #define bitLenInt uint8_t
 #define bitCapInt uint64_t
 #define bitsInByte 8
+#define qrack_rand_gen std::mt19937_64
+#define qrack_rand_gen_ptr std::shared_ptr<std::mt19937_64>
 
 #include "config.h"
 
 #if ENABLE_COMPLEX8
 #include <complex>
 namespace Qrack {
-    typedef std::complex<float> complex;
-    typedef float real1;
-}
+typedef std::complex<float> complex;
```

Comment:
Unrelated change; remove from PR.

Reply:
It's "make format." Do I open up a "make format" PR, or do I "make format" on the next PR? You're not wrong, but that's why this is here.